docs/add-components.md

```diff
@@ -0,0 +1,44 @@
+# Adding components to an ODK repo
+
+To add custom components to an ODK repo, please follow the following steps:
+1. Locate your odk yaml file and open it with your favourite text editor (src/ontology/uberon-odk.yaml)
+2. Search if there is already a component section to the yaml file, if not add it accordingly, adding the name of your component:
+```
+components:
+  products:
+    - filename: your-component-name.owl
+```
+3. Add the component to your catalog file (src/ontology/catalog-v100.xml)
+```
+  <uri name="http://purl.obolibrary.org/obo/uberon/components/your-component-name.owl" uri="components/your-component-name.owl"/>
+```
+4. Add the component to the edit file (src/ontology/uberon-edit.obo)
+for .obo formats: 
+```
+import: http://purl.obolibrary.org/obo/uberon/components/your-component-name.owl
+```
+for .owl formats: 
+```
+Import(<http://purl.obolibrary.org/obo/pcl/components/your-component-name.owl>)
```

Comment:
uberon

Reply:
whoops lol copied over and forgot to change that >.<